docs/graphql/core/guides/environments.rst

```diff
@@ -0,0 +1,268 @@
+.. meta::
+   :description: Guide for moving between environments in Hasura 
+   :keywords: hasura, docs, guide, local dev, staging, production, environment
+
+.. _guide_environments:
+
+Moving between environments
+===========================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 2
+  :local:
+
+Introduction
+------------
+
+This guide will show how to approach various stages of development with Hasura.
```

Comment:
I think this can be written as **This guide talks about the processes on how to approach various stages of development with Hasura.**